adsdocmatch/match_w_metadata.py

```diff
@@ -158,10 +158,10 @@ def match_to_arXiv(self, filename):
                 elif status == 0:
                     return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: do not match.'}]
                 elif status == -1:
-                    return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: did not recognize the bibcode.', 'status_code': 'unrecognizable bibstem, processing stopped, shall be added to the rerun list.'}]
+                    return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: did not recognize the bibcode.', 'status_flaw': 'unrecognizable bibstem, processing stopped, shall be added to the rerun list.'}]
         except Exception as e:
             logger.error('Exception: %s'%e)
-            return [{'source_bibcode': metadata.get('bibcode'), 'comment' : 'Exception: %s in metadata file: %s'%(e, filename), 'status_code' : 'got exception, processing stopped, shall be added to the rerun list.'}]
+            return [{'source_bibcode': metadata.get('bibcode'), 'comment' : 'Exception: %s in metadata file: %s'%(e, filename), 'status_flaw' : 'got exception, processing stopped, shall be added to the rerun list.'}]
```

Comment:
If the input file fails parsing, the variable metadata will be undefined, and the return message itself will generate an exception.

Reply:
Fixed. Thank you.